mobileApp/package.json

```diff
@@ -0,0 +1,41 @@
+{
+  "name": "mobileApp",
+  "version": "0.0.1",
+  "private": true,
+  "scripts": {
+    "android": "react-native run-android",
+    "ios": "react-native run-ios",
+    "start": "react-native start",
+    "test": "jest",
+    "lint": "eslint ."
+  },
+  "devDependencies": {
```

Comment:
Please, read https://stackoverflow.com/questions/18875674/whats-the-difference-between-dependencies-devdependencies-and-peerdependencies and https://qna.habr.com/q/170263